index.html

```diff
@@ -7,8 +7,115 @@
+        </div>
+    </header>
+    <main>
+        <container class="head">
+            <div class="heading-left">
+                <h3 class="launching-soon">LAUNCHING SOON</h3>
+                <h1>
+                    An NFT like <br> no other
+                </h1>
+                <p class="header-text">
+                Don't miss out on the <br> release of our new NFT. Sign <br> up below to receive updates when we go live.
+                </p>    
+                <button class="button">Sign up</button>
+            </div>
+            <div class="heading-right">
+                <img src="graphic.svg" alt="graphic">
+            </div>
+        </container>
+        <container class="background">
+            <div class="background-text">
+                <h2>Free NFT for early birds</h2>
+                <p>Sign up today and you'll get a free NFT when we launch</p>
+            </div>
+        </container>
+        <container class="icons">
+            <div class="icons-main">
+                <div class="icon1">
+                    <img src="1.svg" alt="Boomerang">
+                </div>
+                <div class="icon2">
+                    <img src="2.svg" alt="Blimp">
+                </div>
+                <div class="icon3">
+                    <img src="3.svg" alt="qrco">
+                </div>
+                <div class="icon4">
+                    <img src="4.svg" alt="OpenDoor">
+                </div>
+                <div class="icon5">
+                    <img src="5.svg" alt="Droplry">
+                </div>
+            </div>
+        </container>
+        <container class="NFTs">
+            <div class="NFT-text">
+                <h1>LE Super Rare Auction</h1>
+                <p>We have released four limited edition NFTs which can be bid on via <span class="bold-underline">OpenSea</span>.</p>
+            </div>
+            <div class="NFT-pictures">
+                <div class="box1"><a href="#">
+                    <img src="ntf-1.svg" alt="nft1">
+                    <div class="textbox1">
+                        <h4>Gloop series</h4>
+                        <h1>Purple Man</h1>
+                </div>
+                <div class="box2"><a href="#">
+                    <img src="nft-2.svg" alt="nft2">
+                    <div class="textbox2">
+                        <h4>Gloop series</h4>
+                        <h1>Beige</h1>
+                </div>
+                <div class="box3"><a href="#">
+                    <img src="ntf-3.svg" alt="nft3">
+                    <div class="textbox3">
+                        <h4>Gloop series</h4>
+                        <h1>Red Man</h1>
+                </div>
+                <div class="box4"><a href="#">
+                    <img src="ntf-4.svg" alt="nft4">
+                    <div class="textbox4">
+                        <h4>Gloop series</h4>
+                        <h1>Green</h1>
+                </div>
+            </div>
```

Comment:
Čia didžiausia klaida, kuri sukėlė jums be galo daug problemų - dalinai per prastą formatavimą, dalinai per neapdairumą neuždarėt `<a>` ir `<div>` žymių. To pasekoje `<div>` elementai susidėjo vienas į kitą, dėl to nepasijautė `flex` atributas, o nuoroda taip niekada ir nesibaigė iki dokumento pabaigos.